.github/workflows/add-jira-link-pr-open.yml

```diff
@@ -0,0 +1,83 @@
+#
+# SPDX-FileCopyrightText: 2024 Lifely
+# SPDX-License-Identifier: EUPL-1.2+
+#
+name: Create Jira Link to the PR
+on:
+  pull_request:
+    types:
+      - opened
+
+jobs:
+  create-jira-web-link:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.12'
+
+      - name: Run Python script
+        run: |
+          pip install requests 
+          MY_PYTHON_SCRIPT=$(cat << EOF
```

Comment:
Do we need `MY_PYTHON_SCRIPT`? Can we execute this directly with `python` as we anyway immediately do?